### PR TITLE
Update contributing guide for plugin Strapi v3 release

### PR DESCRIPTION
**Pull request by bidoubiwa**
Update the steps to publish the strapi v3 compatible plugin

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -125,12 +125,17 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
...
 ```javascript
-"version": "X.X.X",
+"version": "X.X.X-strapi-v3.X",
 ```
 
-Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/strapi-plugin-meilisearch/releases): on this page, click on `Edit` (related to the draft release) > update the description (be sure you apply [these recommandations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)) > when you are ready, click on `Publish release`.
+Once the changes are merged on `v3_main`, create a release via the [GitHub interface](https://github.com/meilisearch/strapi-plugin-meilisearch/releases):
+- The title should be `vX.X.X-strapi-v3.X`, same as in `package.json` prefixed by a `v`.
+- The new tag should have the same name as the title.
+- click on `Edit` (related to the draft release) and update the description (be sure you apply [these recommandations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)).
```
Comment (brunoocasali) [inline CONTRIBUTING.md:134]:
```suggestion
- Click on `Edit` (related to the draft release) and update the description (be sure you apply [these recommendations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)).
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "strapi-plugin-meilisearch",
-  "version": "0.6.0-strapi-v3",
+  "version": "0.6.0-strapi-v3.0",
```
Comment (brunoocasali) [inline package.json:3]: Does the file `playground/.tmp/data.db` should be here?

Comment (brunoocasali) [reply package.json:3]: I could not comment in the right place, but there is a file with that name and I'm not sure if it should be committed!

File: CONTRIBUTING.md (hunk 1 above)
Comment (brunoocasali) [reply CONTRIBUTING.md]: Yeah, let's automate it!

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**